doc/source/whatsnew/v1.0.2.rst

```diff
@@ -47,6 +47,33 @@ Bug fixes
 
 .. ---------------------------------------------------------------------------
 
+Indexing with Nullable Boolean Arrays
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Can you move this above bug fixes section, and make this of the same level? (so using `~~~` for underline) 
(it's not a bug fix but a deliberate API change)